lib/adapters/serializers/base.js

```diff
@@ -0,0 +1,15 @@
+var Base = require('extendable-base');
+
+module.exports = Base.extend({
+    initialize: function(stream, options) {
+        this.stream = stream;
+    },
+
+    write: function(points) {
```

Comment:
I think writing to should be the responsibility of the adapter and not the serializer. In that case, this method would be `serialize` and would return a string which the adapter would then write to its stream.